README.md

```diff
@@ -1,19 +1,37 @@
 # svreal
 [![Actions Status](https://github.com/sgherbst/svreal/workflows/Regression/badge.svg)](https://github.com/sgherbst/svreal/actions)
-[![BuildKite Status](https://badge.buildkite.com/45ed712ae8720e9a3e7c040d2e3bc441a18b6ef269e4573723.svg)](https://buildkite.com/stanford-aha/svreal)
+[![BuildKite Status](https://badge.buildkite.com/45ed712ae8720e9a3e7c040d2e3bc441a18b6ef269e4573723.svg?branch=master)](https://buildkite.com/stanford-aha/svreal)
```

Comment:
before this was targeting the latest BuildKite test, not necessarily on the master branch, which was problematic since it meant the front page might show a failing status even if the master branch was passing